src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+2.8.9
+-----
+
+* fixed inheritance of the `autowire` flag
+
```

Comment:
The whole paragraph should be removed as the CHANGELOG is automatically generated.